completion/available/pew.completion.bash

```diff
@@ -2,5 +2,6 @@
 # shellcheck disable=SC1090
```

Comment:
```suggestion
```

If we put `disable=SC1090` on the exact line, I think the global `disable=SC1090` is not needed anymore. Same for `{defaults,travis}.completion.bash`.

Reply:
Fixed both of these in latest commit :+1: